include/config/test.h

```diff
@@ -1129,4 +1129,8 @@
 #undef P_FAMILY_PECHARUNT
 #define P_FAMILY_PECHARUNT               TRUE
 
+// Flags
+#undef B_FLAG_SLEEP_CLAUSE
+#define B_FLAG_SLEEP_CLAUSE              FLAG_SPECIAL_FLAG_UNUSED_0x4003
```

Comment:
Can we somehow handle this without a flag? I assume this is going to break `FLAG_SPECIAL_FLAG_UNUSED_0x4003` for everyone who already uses the flag. 

Sorry, it hit me after I merged.